daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/TransactionCoder.scala

```diff
@@ -256,7 +256,8 @@ object TransactionCoder {
     else {
       val nodeBuilder =
         TransactionOuterClass.Node.newBuilder().setNodeId(encodeNid.asString(nodeId))
-      nodeBuilder.setVersion(node.version.protoValue)
+      if (enclosingVersion >= TransactionVersion.minNodeVersion)
+        nodeBuilder.setVersion(node.version.protoValue)
```

Comment:
I realized that I write the field `version` for Node message even in transaction that does not support it. 
I mean, the node in a transaction V10 should not have field `version`. 
This is not a big deal, as the reader ignore the field in this case -- I add a test to check we continue to ignore it--